cmd/csilvm/csilvm.go

```diff
@@ -81,6 +82,11 @@ func main() {
 	if strings.HasPrefix(sock, "unix://") {
 		sock = sock[len("unix://"):]
 	}
+	// Unlink the domain socket in case it is left lying around from a
+	// previous run. err return is not really interesting because it is
+	// normal for this to fail if the process is starting for the first time.
+	log.Printf("[%s] Unlinking %s", *vgnameF, sock)
```

Comment:
nit: we probably don't need the `[%s]` prefix here (nor in the a priori and subsequent log calls), since it's already a prefix